eox_nelp/pearson_vue/tasks.py

```diff
@@ -0,0 +1,30 @@
+"""
+This module contains functions and classes for making asynchronous calls to Pearson VUE's RTI services.
+
+Functions:
+    real_time_import_task(data: dict) -> None: Performs an asynchronous call to the RTI service.
+"""
+
+from celery import shared_task
+
+from eox_nelp.pearson_vue.rti_backend import RealTimeImport
+
+
+@shared_task(bind=True)
+def real_time_import_task(self, pipeline_index=0, **kwargs):
+    """
+    Performs an asynchronous call to Pearson VUE's RTI (Real Time Import) service.
+
+    This task initiates the real-time import process using the provided pipeline index and optional keyword arguments.
+
+    Args:
+        self: The Celery task instance.
+        pipeline_index (int): The index of the pipeline to be executed (default is 0).
+        **kwargs: Additional keyword arguments to configure the RTI service.
+    """
+    rti = RealTimeImport(pipeline_index=pipeline_index, **kwargs.copy())
+
+    try:
+        rti.run_pipeline()
+    except Exception as exc:  # pylint: disable=broad-exception-caught
+        self.retry(exc=exc, kwargs=rti.backend_data)
```

Comment:
max_retries or there would be a retry policy?
https://docs.celeryq.dev/en/latest/userguide/tasks.html#automatic-retry-for-known-exceptions

Reply:
When the value is None that uses the default value https://docs.celeryq.dev/en/stable/reference/celery.app.task.html#celery.app.task.Task.retry